src/FastSerialization/SegmentedList.cs

```diff
@@ -93,6 +93,8 @@ public int Count
             }
         }
 
+        public int Capacity => this.capacity;
```

Comment:
Would like to understand the need for these two new public APIs.  Are these concerned internally by SegmentedDictionary?